### PR TITLE
Update run task screenshots and docs

### PR DESCRIPTION
**Pull request by hcrhall**
## Labels

- [x] inaccuracy
- [ ] clarification
- [ ] new docs
- [x] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (hcrhall) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:78]: @laurapacilio it would be great to get your take on the following. I'm not sure if this is the best way to phrase this,  and without an official style guide, I am a bit lost. 

I think it might be better to do the following but not sure if it's the right approach.
```
Select the ellipses (...) on the associated run task, and then select "Remove".
```

Comment (laurapacilio) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:78]: hi! Yes I agree this is a tricky one. When our style guide doesn't cover something, we use google developers style guide. This is what it has to say about this: https://developers.google.com/style/ellipses#ellipses-in-a-user-interface

"When ellipses appear in a user interface, exclude them from the documentation describing the user interface unless their omission could cause confusion. For example, if the text on the button in the UI reads Save ..., document it as click Save."

This case is difficult because the button IS ellipsis. I talked to the education team, and we think that your approach with one tweak is the way we want to go. So:

"Click the ellipsis (...) button next to the Run Task, and then select **Remove**."

Also - our style guide now dictates that all UI elements should be bolded, rather than in quotes! Would you mind updating this PR to reflect that? We're slowly going to update all of the Cloud/Enterprise stuff to adhere to the new guideline.

Comment (msboom) [reply content/source/docs/cloud/workspaces/run-tasks.html.md]: Thanks @laurapacilio I have updated the docs as requested. 

I have used `(...)` in the docs the first time that a user interacts with an ellipses button. I have left it out for any subsequent call outs in the docs.

Comment (hcrhall) [reply content/source/docs/cloud/workspaces/run-tasks.html.md]: Thanks @laurapacilio I have updated the docs as requested. 

I have used `(...)` in the docs the first time that a user interacts with an ellipses button. I have left it out for any subsequent call outs in the docs.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (dylanegan) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:25]:
```suggestion
1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
```

Should `click` be lowercase?

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (dylanegan) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:78]: Once we've clicked, are we clicking again or selecting? I think the prior version potentially had `select` or potentially it was a suggestion from @laurapacilio.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (dylanegan) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:87]:
```suggestion
2. On the run task you want to delete, Click the ellipses (...) and then Click **Edit**
```

This introduces the parenthesis here too, as I feel it probably isn't enough to do it once. This suggestion doesn't change `Click` to lowercase nor change it ultimately to `select` though as I'd prefer to keep those as separate discussions.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (hcrhall) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:78]: The developer guide says:

```
When the environment is a desktop with a mouse, use click for most targets, such as buttons, links, list items, and radio buttons
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (hcrhall) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:87]: In another style guide that I referred to, they suggest that you show a visual representation of the term in the first instance and then just use the word in subsequent step. I am not attached to my approach, just what makes sense.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (hcrhall) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:25]: TBH, I am not sure but I do see that we use lower in the SAML docs.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (dylanegan) [reply content/source/docs/cloud/workspaces/run-tasks.html.md]: Works for me.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (dylanegan) [reply content/source/docs/cloud/workspaces/run-tasks.html.md]: Yeah, I guess that works then. Have to keep on top of those first instance in documentation moving forward then. Maybe there's a Markdown hacky way to do it.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:31]:
```suggestion
3. Click **Create run task**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:40]:
```suggestion
5. Click **Create run task**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:46]:
```suggestion
7. Click the **+** next to the task(s) you want to add to the workspace.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:44]:
```suggestion
6. Click **Workspaces** in the top navigation bar, click your workspace in the grid, open the **Settings** menu and select **Run Tasks**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:50]:
```suggestion
8. Choose an enforcement level:
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:53]:
```suggestion
9. Click **Create**. Your run task is now configured.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:56]:
```suggestion

```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:51]:
```suggestion
  - **Advisory**  - Run tasks can not block a run from completing. If the task fails, the run will proceed with a warning in the UI.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:52]:
```suggestion
  - **Mandatory** - Run tasks can block a run from completing. If the task fails (including a timeout or unexpected remote error condition), the run will transition to an Errored state with a warning in the UI.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:62]:
```suggestion
Run tasks perform actions between the [plan](../run/states.html#2-the-plan-stage) and [apply](../run/states.html#6-the-apply-stage) stages of a [Terraform run](../run/index.html). Once all run tasks have completed, the run will end based on the most restrictive enforcement level in each associated run task.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:64]:
```suggestion
For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, Terraform Cloud displays the status and any related message data in the UI.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:76]:
```suggestion
Removing a run task from a workspace does not delete it from the organization. To remove a run task from a specific workspace: 

1. Navigate to the desired workspace, open the **Settings** menu and select **Run Tasks**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:78]: It's just nice to give users one sentence that confirms what they just did and lets them know the result. 

```suggestion
2. Click the ellipses (...) on the associated run task, and then click **Remove**. The run task will no longer be applied to runs within the workspace.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:85]: Are we in the main nav bar again? Should we specify?

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:87]:
```suggestion
2. Click the ellipses (...) next to the run task you want to delete, and then click **Edit**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:89]:
```suggestion
3. Click **Delete run task**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:93]: I would move this up to come right after **Delete** and before the image showing the warning.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:95]: I think we should move this line to come right before the instructions in this section.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:85]:
```suggestion
You must remove a run task from all associated workspaces before you can delete it. To delete a run task:

1. Navigate to **Settings** in the top navigation bar and click **Run Tasks**.
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:23]: This matches the language on all of our buttons, etc. a bit better! 

```suggestion
## Creating a Run Task
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:35]: Just make sure that when you build locally this renders properly :-) Sometimes the renderer can be finicky about nesting bullets underneath numbered steps.

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/workspaces/run-tasks.html.md:15]: Avoiding adding another note - we want to use notes as sparingly as possible. Also, we want to be inclusive and not use "see". So, I replaced it  - lmk what you think! 

```suggestion
You can manage run tasks through the UI as detailed below or through the [Run Tasks APIs](../api/run-tasks.html).
```

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (laurapacilio) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:44]: Should this be the start of a new section called "Adding Run Tasks to a Workspace"?  It just seems like we've jumped from creating it to a new thing that users may want to do.

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block you! Aside from these comments, I know we've been talking about screenshots - I know that the information in the README doesn't match the guidelines for Learn that I sent you, so I can open a separate PR to adjust screenshots after we merge the initial content if that's helpful!

The other thing is - I made a lot of edits through GitHub UI and sometimes the formatting gets a bit weird when you apply them this way. Just make sure that you build the site locally and preview the changes again before merging to make sure everything looks okay. You can find instructions about how to do that here: https://github.com/hashicorp/terraform-website#running-the-site-locally

Thank you so much for taking the time to update this content  :-)

**Review by dylanegan — APPROVED** [review]

File: content/source/docs/cloud/workspaces/run-tasks.html.md
Comment (hcrhall) [reply content/source/docs/cloud/workspaces/run-tasks.html.md:13]: Sounds good.

**Pull request merged**
